src/coreclr/vm/i386/asmhelpers.S

```diff
@@ -663,47 +663,6 @@ LOCAL_LABEL(GoCallCalliWorker):
     jmp C_FUNC(GenericPInvokeCalliHelper)
 LEAF_END GenericPInvokeCalliHelper, _TEXT
 
-// ==========================================================================
-NESTED_ENTRY ExternalMethodFixupStub, _TEXT_ NoHandler
-    // pop off the return address to the stub
-    // leaving the actual caller's return address on top of the stack
-    pop     eax
-
-    STUB_PROLOG
-
-    mov         esi, esp
-
-    // EAX is return address into CORCOMPILE_EXTERNAL_METHOD_THUNK. Subtract 5 to get start address.
-    sub         eax, 5
-
-.att_syntax
-    pushl       $0
-    pushl       $0
-.intel_syntax noprefix
-
-    push        eax
-
-    // pTransitionBlock
-    push        esi
-
-    call        C_FUNC(ExternalMethodFixupWorker)
-
-    // eax now contains replacement stub. PreStubWorker will never return
-    // NULL (it throws an exception if stub creation fails.)
-
-    // From here on, mustn't trash eax
-
-    STUB_EPILOG
-
-PATCH_LABEL ExternalMethodFixupPatchLabel
```

Comment:
Should this be deleted this as well:
https://github.com/dotnet/runtime/blob/66eb69ed282c2d35fc64f94e76cf3eeaf83edb5c/src/coreclr/vm/i386/asmhelpers.asm#L1290-L1291